core/payment/src/alloc_release_task.rs

```diff
@@ -0,0 +1,44 @@
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+use std::sync::Arc;
+
+#[derive(Debug, Clone)]
+pub struct AllocationReleaseTasks {
+    pub tasks: Arc<parking_lot::Mutex<HashMap<String, tokio::task::JoinHandle<()>>>>,
```

Comment:
Whats wrong with std lock, that we need parking lot?

Reply:
more elegant no unwrap